app/forms/declaration/Document.scala

```diff
@@ -18,13 +18,15 @@ package forms.declaration
 
 import forms.DeclarationPage
 import forms.Mapping.requiredRadio
+import models.declaration.DocumentCategory
+import models.declaration.DocumentCategory.{RelatedDocument, SimplifiedDeclaration}
 import play.api.data.Forms.{optional, text}
 import play.api.data.{Form, FormError, Forms}
 import play.api.libs.json.{JsValue, Json}
 import services.DocumentType
 import utils.validators.forms.FieldValidator._
 
-case class Document(documentCategory: String, documentType: String, documentReference: String, goodsItemIdentifier: Option[String]) {
+case class Document(documentType: String, documentReference: String, documentCategory: DocumentCategory, goodsItemIdentifier: Option[String]) {
```

Comment:
Re-ordering mapping/constructor params to match field order on form.  Also decided to create DocumentCategory as a type.